msteams-platform/tabs/how-to/build-adaptive-card-tabs.md

```diff
@@ -6,36 +6,38 @@ ms.topic: conceptual
 ms.author: surbhigupta
 ---
 
-
 # Build tabs with Adaptive Cards
 
 > [!IMPORTANT]
 > * This feature is in [Public Developer Preview](~/resources/dev-preview/developer-preview-intro.md) and is supported in desktop and mobile. Support in the web browser is coming soon.
 > * Tabs with Adaptive Cards are currently only supported as personal apps.
 
-Use Adaptive Cards to build tabs with ease. You can build your tabs with ready-made UI Lego-blocks that look and feel native on desktop, web, and mobile. Building tabs with Adaptive Cards centralizes all Teams app capabilities around a bot backend and Adaptive Card frontend, thus, eliminating the need for a different backend for your bot and tabs. This greatly reduces server and maintenance costs of your Teams app. This article helps you understand the changes required to be made to the app manifest, how the invoke activity requests and sends information in tab with Adaptive Cards, and the impact on the task module workflow. 
+When developing a tab using the traditional method, you must consider these issues, such as HTML and Cascading Style Sheets (CSS) considerations, slow load times, iFrame constraints, and server maintenance and costs. Adaptive Card tabs is a new way to build tabs in Teams. Instead of embedding web content in an IFrame, you can render Adaptive Card to a tab. While the front-end is rendered as Adaptive Card, the backend is powered by a bot. The bot is responsible for accepting requests and responding appropriately with the Adaptive Card that is rendered.
```

Comment:
1. This sentence reads unintuitively. Consider something like:
```
When developing a tab using the traditional method, you may run into these issues: ...."
```
2. No one writes out CSS as "Cascading Style Sheets". Can we just say "CSS"?
3. "You can render Adaptive Card**s** to a tab"
4. "While the front-end is rendered with Adaptive Card**s**"